src/ucs/stats/stats.c

```diff
@@ -460,6 +465,138 @@ void ucs_stats_node_free(ucs_stats_node_t *node)
     }
 }
 
+static inline uint32_t ucs_stats_class_id_to_aggregate_sum_base_offset_get(
```

Comment:
why does it need to be inline?

Reply:
Doesn't have to even though it's called only once in the code. Removed the inline.